projects/storefrontlib/src/lib/cms-lib/address-book/address-book.component.spec.ts

```diff
@@ -144,48 +147,12 @@ describe('AddressBookComponent', () => {
     expect(el.queryAll(By.css('cx-address-card')).length).toEqual(3);
   });
 
-  it('should show and hide add address form', () => {
-    el.query(By.css('.btn-action')).nativeElement.click();
-    fixture.detectChanges();
-    expect(component.showAddAddressForm).toBe(true);
-    checkSectionMsg('Add');
-
-    fixture.whenStable().then(() => {
-      el.query(
-        By.css('.cx-address-form__btns .btn-action')
-      ).nativeElement.click();
-      fixture.detectChanges();
-      checkSectionMsg('Saved');
-    });
-  });
-
-  it('should show and hide edit address form', () => {
-    fixture.whenStable().then(() => {
-      el.query(By.css('.cx-address-card__actions .edit')).nativeElement.click();
-      fixture.detectChanges();
-      expect(component.showEditAddressForm).toBe(true);
-      checkSectionMsg('Edit');
-
-      el.query(
-        By.css('.cx-address-form__btns .btn-action')
-      ).nativeElement.click();
-      fixture.detectChanges();
-      checkSectionMsg('Saved');
-    });
+  it('should display the saved shipping addresses', () => {
+    checkSectionMsg('Saved shipping addresses');
```

Comment:
if we use this function only once here, let's drop it and use the logic right here

Reply:
good point, refactored